Update elasticsuite-tracking-indices-causes-problems-with-elasticsear… (#140)
magento/knowledge-base: src/troubleshooting/general/elasticsuite-tracking-indices-causes-problems-with-elasticsearch.md

## Solution

> ### Upgrade your Elasticsuite version to be able to cleanup tracking indices periodically

Reviewer: ```suggestion
### Upgrade your ElasticSuite version to be able to cleanup tracking indices periodically
```

> ### Upgrade your ElasticSuite version to be able to cleanup tracking indices periodically

Reviewer 2: ```suggestion
### Upgrade your ElasticSuite version to be able to clean up tracking indices periodically
```

Once you have upgraded the ElasticSuite plugin to version > 2.8.0, you can configure a periodical cleaning of indices. 

Go to **Stores > Configuration > Tracking > Global Configuration > Retention Delay**

The default retention period is 365 days. You can reduce it to 30 or 15 days.

> ### Upgrade your Elasticsuite version to use monthly based indices instead of daily based indices

Reviewer: ```suggestion
### Upgrade your ElasticSuite version to use monthly based indices instead of daily based indices
```
